api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirMedicationRequestServiceImpl.java

```diff
@@ -29,7 +35,7 @@
 	private FhirMedicationRequestDao dao;
 	
 	@Override
-	public MedicationRequest getMedicationRequestByUuid(String uuid) {
-		return medicationRequestTranslator.toFhirResource(dao.getMedicationRequestByUuid(uuid));
+	protected OpenmrsFhirTranslator<DrugOrder, MedicationRequest> getTranslator() {
```

Comment:
You shouldn't need to define this method. Just rename `medicationRequestTranslator` to `translator`

Reply:
i have removed it to